lang/languages-nl.json

```diff
@@ -1,27 +1,38 @@
 {
-    "en": "",
-    "af": "",
-    "az": "",
-    "bg": "",
-    "cs": "",
-    "de": "",
-    "el": "",
-    "eo": "",
-    "es": "",
-    "fr": "",
-    "hy": "",
-    "it": "",
-    "ja": "",
-    "ko": "",
-    "nb": "",
-    "oc": "",
-    "pl": "",
-    "ptBR": "",
-    "ru": "",
-    "sk": "",
-    "sl": "",
-    "sv": "",
-    "tr": "",
-    "vi": "",
-    "zhCN": ""
-}
\ No newline at end of file
+    "en": "Engels",
+    "af": "Afrikaans",
+    "az": "Azerbeidzjaans",
+    "bg": "Bulgaars",
+    "ca": "Catalaans",
+    "cs": "Tsjechisch",
+    "da": "Deens",
+    "de": "Duits",
+    "enGB": "English (Verenigd Koninkrijk)",
```

Comment:
Engels I presume?

Reply:
Oops, inderdaad. :-)

Reply:
Would you like to fix it, or should I submit a patch?

Reply:
I'm no Jitsi maintainer, just some Dutch guy trying to help ;-)
So please update this PR.

Reply:
This other Dutch guy thanks you for that!
I've pushed the fix.